src/main/java/by/exadel/internship/exception_handing/GlobalExceptionHandler.java

```diff
@@ -25,6 +25,13 @@
         return new ResponseEntity<>(data, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler
+    public ResponseEntity<IncorrectData> handleException(
+            InappropriateRoleException exception) {
```

Comment:
Remove it. Inappropriate Role Exception means bad request and is handled by correct handler. It's excessive.

Reply:
Done